pkg/minikube/config/types.go

```diff
@@ -70,3 +73,9 @@ type KubernetesConfig struct {
 
 	ShouldLoadCachedImages bool
 }
+
+// KubernetesSemVer gives us the semver object corresponding to the requested
+// Kubernetes version or panics.
+func (kcfg KubernetesConfig) KubernetesSemVer() semver.Version {
+	return semver.MustParse(strings.TrimPrefix(kcfg.KubernetesVersion, version.VersionPrefix))
```

Comment:
Libraries should not panic: https://github.com/golang/go/wiki/CodeReviewComments#dont-panic